code/src/components/Form.js

```diff
@@ -0,0 +1,153 @@
+import React, { useState } from 'react'
+
+import Intro from './Intro'
+import QuestionOne from './QuestionOne'
+import QuestionTwo from './QuestionTwo'
+import QuestionThree from './QuestionThree'
+import Summary from './Summary'
+import ButtonNext from './ButtonNext'
+import ButtonBack from './ButtonBack'
+
+const flowers = ['roses', 'tulips', 'orchids']
+
+const Form = () => {
+ 
+    const [name, setName] = useState('');
+    const [size, setSize] = useState('')
+    const [flower, setFlower] = useState('')
+    const [questionIndex, setQuestionIndex] = useState(0)
+
+
+
+    // const onBackIndexChange = () => {
+    //     setQuestionIndex(questionIndex -1)
+    // }
+
+
+    console.log(size, size, size)
+    console.log(name, name, name)
+    console.log(flower, flower, flower)
+    console.log(questionIndex)
+
+    return (
+        <>
+            <form onSubmit={event => event.preventDefault()}>
+                
+                {/* INTRO SECTION */}
+                {questionIndex === 0 && 
+                    <>
+                        <section aria-label="Intro page" className='intro-page'>
+                            <Intro/> 
+                            <ButtonNext
+                                questionIndex={questionIndex}
+                                setQuestionIndex={setQuestionIndex}
+                            />
```

Comment:
Like I wrote above about the type="submit". Then you could have used the function in onSubmit instead. Like <form onSubmit={onQuestionChangeIndex}, and also moved preventDefault into that function. And then just had <ButtonNext /> :D